.eslintrc.yml

```diff
@@ -1,9 +1,11 @@
-extends: "@hamster-bot/eslint-config/typescript"
-
+extends:
+  - '@hydrooj/eslint-config/base.yaml'
```

Comment:
There are hundreds of rules that controls code style.
(`array-bracket-spacing`, `block-spacing`, `brace-style`, `camelcase`, `comma-dangle`, `comma-spacing`, `comma-style`, `computed-property-spacing`, ...)
Unfortunately, `eslint-config-recommended` introduces only little of them.
It would be much easier (and makes more sense) to configure using a strict preset than writing all config by hand.